src/main/kotlin/uk/gov/dluhc/printapi/rest/SecurityController.kt

```diff
@@ -1,23 +0,0 @@
-package uk.gov.dluhc.printapi.rest
```

Comment:
I've deleted this class and it's test because they are no longer needed now we have other REST APIs that use and test the security. This class should be been deleted a while back. Hey ho 👍